public/assets/jquery-ui/effect-puff.js

```diff
@@ -38,7 +38,7 @@ function(t){"function"==typeof define&&define.amd?define(["jquery","./effect"],t
  *
  * http://api.jqueryui.com/scale-effect/
  */
-function(t){"function"==typeof define&&define.amd?define(["jquery","./effect","./effect-size"],t):t(jQuery)}(function(t){return t.effects.effect.scale=function(e,n){var r=t(this),o=t.extend(!0,{},e),i=t.effects.setMode(r,e.mode||"effect"),a=parseInt(e.percent,10)||(0===parseInt(e.percent,10)?0:"hide"===i?0:100),s=e.direction||"both",f=e.origin,u={height:r.height(),width:r.width(),outerHeight:r.outerHeight(),outerWidth:r.outerWidth()},c={y:"horizontal"!==s?a/100:1,x:"vertical"!==s?a/100:1};o.effect="size",o.queue=!1,o.complete=n,"effect"!==i&&(o.origin=f||["middle","center"],o.restore=!0),o.from=e.from||("show"===i?{height:0,width:0,outerHeight:0,outerWidth:0}:u),o.to={height:u.height*c.y,width:u.width*c.x,outerHeight:u.outerHeight*c.y,outerWidth:u.outerWidth*c.x},o.fade&&("show"===i&&(o.from.opacity=0,o.to.opacity=1),"hide"===i&&(o.from.opacity=1,o.to.opacity=0)),r.effect(o)}}),/*!
+function(t){"function"==typeof define&&define.amd?define(["jquery","./effect","./effect-size"],t):t(jQuery)}(function(t){return t.effects.effect.scale=function(e,i){var n=t(this),o=t.extend(!0,{},e),r=t.effects.setMode(n,e.mode||"effect"),s=parseInt(e.percent,10)||(0===parseInt(e.percent,10)?0:"hide"===r?0:100),a=e.direction||"both",l=e.origin,u={height:n.height(),width:n.width(),outerHeight:n.outerHeight(),outerWidth:n.outerWidth()},c={y:"horizontal"!==a?s/100:1,x:"vertical"!==a?s/100:1};o.effect="size",o.queue=!1,o.complete=i,"effect"!==r&&(o.origin=l||["middle","center"],o.restore=!0),o.from=e.from||("show"===r?{height:0,width:0,outerHeight:0,outerWidth:0}:u),o.to={height:u.height*c.y,width:u.width*c.x,outerHeight:u.outerHeight*c.y,outerWidth:u.outerWidth*c.x},o.fade&&("show"===r&&(o.from.opacity=0,o.to.opacity=1),"hide"===r&&(o.from.opacity=1,o.to.opacity=0)),n.effect(o)}}),/*!
```

Comment:
__[jshint]__
* Expected an assignment or function call and instead saw an expression.
* Did you mean to return a conditional instead of an assignment?
* Too many errors. (80% scanned).

<!-- Comment by [SideCI](https://sideci.com) -->